presenters.md

```diff
@@ -126,4 +163,7 @@ titleDisplay: "Presenter"
       </div>
   </div>
   <input type="submit" name="submit" value="Submit session update" class="button mt-1 mb-2">
+  <input type="hidden" name="_format" value="plain">
```

Comment:
Are these hidden fields needed now that we aren't using the third party service?